src/capi/topology.cpp

```diff
@@ -230,6 +230,39 @@ extern "C" chfl_status chfl_topology_residues_linked(
     )
 }
 
+extern "C" chfl_status chfl_topology_add_bond_order(CHFL_TOPOLOGY* const topology, uint64_t i, uint64_t j, chfl_bond_order bo) {
+    CHECK_POINTER(topology);
+    CHFL_ERROR_CATCH(
+        topology->add_bond(checked_cast(i), checked_cast(j), static_cast<Connectivity::BondOrder>(bo));
+    )
+}
+
+#pragma intel optimization_level 2  /* Using -O3 with icc lead to partial copy of the bonds */
```

Comment:
Is this needed here too ?

Reply:
I'm not sure actually. I figured the code was similar to the other functions so the `pragma` would be appropriate.

Reply:
Ok. I'll do some tests to see if this is still needed nowadays.